openquakeplatform/test/tilestream_test.py

```diff
@@ -28,71 +28,84 @@ class TilestreamTest(unittest.TestCase):
-
-        pla.xpath_finduniq("//div[text() = '1942-02-09T07:11:00']", timeout=50)
-
-        # return content GETJson in Tilestream Source.js
-        s = open(
-                "/home/ubuntu/oq-platform2/openquakeplatform/"
-                "static/js/TileStreamSource.js").read()
-        s = s.replace('TilesetT', 'Tileset')
-        r = open(
-                "/home/ubuntu/oq-platform2/openquakeplatform/"
-                "static/js/TileStreamSource.js", 'w')
-        r.write(s)
-        r.close()
-
-        restart_apache()
+        try:
+            # change content GETJson in Tilestream Source.js
+            s = open(
+                    "/home/ubuntu/oq-platform2/openquakeplatform/"
+                    "static/js/TileStreamSource.js").read()
+            s = s.replace('Tileset', 'Tileset_wrong')
+            r = open(
+                    "/home/ubuntu/oq-platform2/openquakeplatform/"
+                    "static/js/TileStreamSource.js", 'w')
+            r.write(s)
+            r.close()
+
+            restart_apache()
+
+            pla = platform_get()
+
+            pla.get('/explore')
+            pla.wait_new_page("//b[contains(text(),"
+                              " 'Seismic Hazard Data Sets and Models')]",
+                              "/explore", strategy="next", timeout=15)
+
+            enter_button = pla.xpath_finduniq(
+                "//li/a[normalize-space(text()) = 'Global "
+                "Instrumental Earthquake Catalogue (1900 - 2009)']")
+
+            href = enter_button.get_attribute('href')
+            id_map = os.path.basename(href)
+            enter_button.click()
+
+            enter_button_view = pla.xpath_finduniq(
+                "//a[@href='/maps/%s/view' and "
+                "normalize-space(text()) = 'View Map']" % id_map)
+            enter_button_view.click()
+
+            enter_button_ident = pla.xpath_finduniq(
+                "//button[@type='button' and normalize-space(text())"
+                "= 'Identify']", timeout=20)
+            enter_button_ident.click()
+
+            # wait info button will be clicked
+            pla.xpath_finduniq(
+                "//button[@type='button' and normalize-space(text())"
+                "= 'Identify']/../../../../..[contains(concat(' ', @class, ' '),"
+                " ' x-btn-pressed ')]", timeout=20)
+
+            _, x, y = pla.xpath_finduniq_coords(
+                "//img[contains(@src, 'LAYERS=oqplatform%3Aisc_viewer_measure"
+                "&FORMAT=image%2Fpng&TRANSPARENT=TRUE&SERVICE=WMS&VERSION="
+                "1.1.1&REQUEST=GetMap&STYLES=&TILED=true&SRS=EPSG%3A900913&"
+                "BBOX=-5009377.085,0,0,5009377.085&WIDTH=256&HEIGHT=256')]",
+                timeout=50)
+
+            pla.add_click_event()
+            pla.click_at(107 + x, 70 + y)
+
+            pla.xpath_finduniq("//div[text() = '1942-02-09T07:11:00']", timeout=50)
+
+            # return content GETJson in Tilestream Source.js
+            s = open(
```

Comment:
Isn't possible to avoid absolute paths ? I think that there is some alternative like some env variable that include the first part of the path.